src/effects/VST3/VST3OptionsDialog.cpp

```diff
@@ -100,6 +102,17 @@ void VST3OptionsDialog::PopulateOrExchange(ShuttleGui & S)
                           mUseGUI);
          }
          S.EndStatic();
+         S.StartStatic(XO(""));
+         {
+            S.AddVariableText(XO(
+"Don't list MIDI CC parameters in plain UI mode."
```

Comment:
What does "CC" abbreviate?  I still don't know.  Would an uninstructed user know?

Reply:
I'll put that into quotes, in that case "MIDI CC" relates to parameter title rather than a term.
On the other hand, considering that we don't support VSTi yet, it might be better remove that option at all. @LWinterberg

Reply:
I'd say you can safely hide all the MIDI stuff in plainUI mode for the time being, yeah.